docs/data/material/varnish/social-share-buttons/social-share-buttons.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Social Share Buttons
+githubLabel: 'varnish: social-share-buttons'
+---
+
+# Social Share Buttons
```

Comment:
was planning to add in a separate PR because those were related to the skiff-template issue of having all social share buttons; i was doing this issue-by-issue but i can add it to this one, either is fine with me!

Reply:
let me merge this and add as a follow-up; want to get this resolved with the other docs PR, was planning to add these buttons for the skiff-template task separately anyways